enhancements/installer/crc.md

```diff
@@ -0,0 +1,93 @@
+---
+title: code-ready-containers
+authors:
+reviewers:
+approvers:
+creation-date: 2020-03-02
+last-updated: 2020-03-02
+status: implemented
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+superseded-by:
+---
+
+# Code Ready Containers (CRC)
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance, 
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this? 
+
+## Summary
+
+CRC functions by doing roughly....
+ 1.  Running a libvirt install, which is for ["development only"](https://github.com/openshift/installer/blob/ffc34e32fe4a71560f59312384daa87b401d6ec9/README.md) meaning OpenShift development only.
+ 2.  On a single node cluster, which is below the minimum of [three that are required](https://github.com/openshift/installer/blob/c904277e59dd947a8884265b2511034b05c38644/upi/openstack/inventory.yaml#L35) for a functional cluster.
+ 3.  Disable the CVO, which prevents all reconciliation of operators, top level status, and upgrades.
```

Comment:
We've actually just merged https://github.com/code-ready/snc/pull/157 to avoid disabling the CVO altogether

Reply:
I'm not sure what do you gain by enabling CVO. As @eparis mentioned, upgrades are not supported and you apply overrides to some operators, which basically make CVO useless ?

Reply:
This makes us one step closer to a real OpenShift cluster as we disable operators through something provided by OpenShift rather than doing it manually.
And more importantly, the CVO does more than just upgrades, among other things it takes care of propagating proxy settings when `inject-proxy` is set. It's better to let it take care of it rather than doing this manually in crc.